packages/react/src/components/MultiSelect/next/FilterableMultiSelect.js

```diff
@@ -165,6 +166,10 @@ const FilterableMultiSelect = React.forwardRef(function FilterableMultiSelect(
   }
 
   function handleOnInputValueChange(inputValue, { type }) {
+    if (onInputValueChange) {
+      onInputValueChange(inputValue, { type });
```

Comment:
One quick question, do you know if you need the `type` parameter here for your use case?  Just was wondering if we could drop that in case we ever need to swap out Downshift in the future

Reply:
@joshblack i thought about that and just thought,.. eh lets keep it up to the user to have full access